torch/_inductor/graph.py

```diff
@@ -938,9 +938,9 @@ def compile_to_module(self):
 
         # Logged twice as per https://github.com/pytorch/pytorch/pull/99038#discussion_r1167826029
         # TODO. Revisit this once the logging API is more mature
-        output_code_log.info("Output code written to: %s", mod.__file__)
         log.debug("Output code written to: %s", mod.__file__)
         output_code_log.debug("Output code: \n%s", code)
+        output_code_log.info("Output code written to: %s", mod.__file__)
```

Comment:
?

Reply:
i actually had this change locally too lol, it makes it nicer to find the file with TORCH_COMPILE_DEBUG